exasol/ds/sandbox/lib/dss_docker/create_image.py

```diff
@@ -175,7 +183,8 @@ def _commit_container(
             "Env": [
                 f"VIRTUAL_ENV={virtualenv}",
                 f"NOTEBOOK_FOLDER_FINAL={notebook_folder_final}",
-                f"NOTEBOOK_FOLDER_INITIAL={notebook_folder_initial}"
+                f"NOTEBOOK_FOLDER_INITIAL={notebook_folder_initial}",
+                f"PATH={path}",
```

Comment:
This can not go here, because the we can not activate the the environment for the whole system. It would be also for the wrong user. This needs to go into the entry point because there we are calling the correct python version and only there this addition to PATH is valid.

Reply:
We actually need to do the same for the AMI and VMs, this means we should use the entry point in the systemd unit. We already have a ticket for that. Please, work on that afterwards.

Reply:
Ok, see next push.